RecoCTPPS/PixelLocal/interface/RPixDetTrackFinder.h

```diff
@@ -0,0 +1,61 @@
+/*
+ *
+* This is a part of CTPPS offline software.
+* Author:
+*   Fabrizio Ferro (ferro@ge.infn.it)
+*   Enrico Robutti (robutti@ge.infn.it)
+*   Fabio Ravera   (fabio.ravera@cern.ch)
+*
+*/
+#ifndef RecoCTPPS_PixelLocal_RPixDetTrackFinder_H
+#define RecoCTPPS_PixelLocal_RPixDetTrackFinder_H
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/CTPPSDetId/interface/CTPPSPixelDetId.h"
+#include "DataFormats/CTPPSReco/interface/CTPPSPixelLocalTrack.h"
+#include "RecoCTPPS/PixelLocal/interface/RPixDetPatternFinder.h"
+
+
+#include "CLHEP/Vector/ThreeVector.h"
+
+#include <vector>
+#include <map>
+
+class RPixDetTrackFinder{
+
+	public:
+		RPixDetTrackFinder(edm::ParameterSet const& parameterSet): parameterSet_(parameterSet), romanPotId_(CTPPSPixelDetId(0, 2, 3, 0)) {}
+		//romanPotId_ is needed to be defined in order to 
+
+		virtual ~RPixDetTrackFinder();
+
+		void setHits(const std::map<CTPPSPixelDetId, std::vector<RPixDetPatternFinder::PointInPlane> > hitMap) {hitMap_ = hitMap; }
```

Comment:
similar to RPixDetPatternFinder: please justify why full copies of input collections and conditions are needed.
if actually needed, replace all copies by value to use by reference.